examples/tagmap/deckgl-overlay.js

```diff
@@ -0,0 +1,38 @@
+import React, {Component} from 'react';
+import DeckGL from 'deck.gl';
+import TagmapLayer from './tagmap-layer';
+
+export default class DeckGLOverlay extends Component {
+
+  static get defaultViewport() {
+    return {
+      latitude: 39.10,
+      longitude: -94.57,
+      zoom: 3.8,
+      maxZoom: 16,
+      pitch: 0,
+      bearing: 0
+    };
+  }
+
+  _initialize(gl) {
+    gl.blendFuncSeparate(gl.SRC_ALPHA, gl.ONE, gl.ONE_MINUS_DST_ALPHA, gl.ONE);
+    gl.blendEquation(gl.FUNC_ADD);
+  }
```

Comment:
We probably should use our latest luma.gl state management APIs here, as shown in https://github.com/uber/deck.gl/commit/303be1dcc2c0706eb55b48f0ad8cb297b80892e2#diff-290511352403f2b78adea66edf5e79acR85